src/ui/widgets/EmbeddedDisplay/embeddedDisplay.tsx

```diff
@@ -106,21 +118,12 @@ export const EmbeddedDisplay = (
       props.position.width = props.position.width || description.width;
     }
 
-    // Overflow set to scroll only if needed
-    // If height or width is defined and is smaller than Bob
-    const overflow =
-      props.position instanceof AbsolutePosition &&
-      (description.height > (props.position.height || 0) ||
-        description.width > (props.position.width || 0))
-        ? "scroll"
-        : "visible";
-
     component = widgetDescriptionToComponent({
       type: "display",
       position: props.position,
       border:
         props.border ?? new Border(BorderStyle.Line, new Color("lightgrey"), 1),
-      overflow: overflow,
+      overflow: props.scroll ?? props.scroll ? "scroll" : "visible",
```

Comment:
I may be missing something but can this not be simplified to:
`props.scroll ? "scroll": "visible"`, I don't think we need the ??.